### DISCUSSION TITLE
How do I use cargo-multivers with Criterion?

### DISCUSSION THREAD

**Discussion by cfkaran2** [ronnychevalier/cargo-multivers#3] [Q&A] [answered]
I use [`cargo criterion`](https://crates.io/crates/criterion) to benchmark my code.  I would like to use it with [`cargo multivers`](https://crates.io/crates/cargo-multivers), but I'm not sure how to do this.  Can someone please explain it to me?

**Comment by ronnychevalier** [comment, marked as answer]
You cannot use `cargo multivers` with `cargo criterion` for the moment.
The current version of `cargo multivers` only has the ability to build a binary crate from a package.
Building a specific binary from a package or a building a benchmark from a package, are not supported use cases yet.

Feel free to add an issue or submit a PR to improve the support of this use case!

**Reply by cfkaran2** [reply]
Thank you, I'll go ahead and do so.  And thank you for making this tool, I expect that it will improve my code's performance significantly (and I'm hoping that I'll be able to prove that via benchmarks in the future!).
